commcare_connect/opportunity/tasks.py

```diff
@@ -73,3 +80,44 @@ def generate_payment_export(opportunity_id: int, export_format: str):
         content = content.encode()
     default_storage.save(export_tmp_name, ContentFile(content))
     return export_tmp_name
+
+
+@celery_app.task()
+def send_notification_inactive_users(opportunity_id: int):
+    opportunity = Opportunity.objects.get(opportunity_id=opportunity_id)
+    opportunity_accesses = OpportunityAccess.objects.filter(opportunity_id=opportunity_id)
+    for access in opportunity_accesses:
+        last_user_learn_module = (
+            CompletedModule.objects.filter(user=access.user, opportunity_id=opportunity_id).order_by("date").last()
+        )
+        has_started_deliver = OpportunityClaim.objects.exists(access=access)
+        if (
+            last_user_learn_module.date <= datetime.date.today() - datetime.timedelta(days=3)
+            and not has_started_deliver
+        ):
+            message = Message(
+                usernames=[access.user.username],
+                title=gettext(f"Resume your learning journey for {opportunity.name}"),
+                body=gettext(
+                    f"You have not completed your learning for {opportunity.name}."
+                    "Please complete the learning modules to start delivering visits."
+                ),
+            )
+            send_message(message)
```

Comment:
Should there be a `continue` after this? I think the rest of the loop isn't relevant if we're in this `if` branch.